src/components/SearchEngineOptimization/SearchEngineOptimization.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+  <div class="search-engine-optimization">
+    <FormField
+      v-model="title"
```

Comment:
you don't need v-model here
just pass value from props and listen change event
the same for other fields